src/Fixer/DoctrineAnnotation/DoctrineAnnotationSpacesFixer.php

```diff
@@ -0,0 +1,247 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\DoctrineAnnotation;
+
+use Doctrine\Common\Annotations\DocLexer;
+use PhpCsFixer\AbstractDoctrineAnnotationFixer;
+use PhpCsFixer\ConfigurationException\InvalidFixerConfigurationException;
+use PhpCsFixer\Doctrine\Annotation\Token;
+use PhpCsFixer\Doctrine\Annotation\Tokens;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+
+/**
+ * Fixes spaces around commas and assignment operators in Doctrine annotations.
+ */
+final class DoctrineAnnotationSpacesFixer extends AbstractDoctrineAnnotationFixer
+{
+    /**
+     * @var bool
+     */
+    private $aroundParentheses;
+
+    /**
+     * @var bool
+     */
+    private $aroundCommas;
+
+    /**
+     * @var bool
+     */
+    private $aroundAssignments;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configure(array $configuration = null)
+    {
+        if (null === $configuration) {
+            $this->aroundParentheses = true;
+            $this->aroundCommas = true;
+            $this->aroundAssignments = true;
+
+            parent::configure(null);
+
+            return;
+        }
+
+        if (!count($configuration)) {
+            parent::configure($configuration);
+
+            return;
+        }
+
+        $configuration = array_merge(array(
+            'around_parentheses' => true,
+            'around_commas' => true,
+            'around_assignments' => true,
+        ), $configuration);
+
+        $this->aroundParentheses = (bool) $configuration['around_parentheses'];
```

Comment:
please take a look at other configurable fixers.
if setting is not a bool, raise an exception. if key is not there, use default value.